layouts/IndexLayout.tsx

```diff
@@ -24,8 +24,8 @@ const IndexLayout: FC<PropsWithChildren> = ({ children }) => {
             {labels['download']}
           </h2>
 
-          <HomeDownloadButton {...currentLtsVersion!} />
-          <HomeDownloadButton {...currentNodeVersion!} />
+          <HomeDownloadButton nodeReleaseData={lts} />
```

Comment:
Since `lts` and `current` are nullable, I rather you do:

```suggestion
          {lts && <HomeDownloadButton nodeReleaseData={lts} />}
```

Reply:
Oops, I completely forget this syntax exists, thank you!

Reply:
Let me know if this worked <3